saf.spec

```diff
@@ -0,0 +1,51 @@
+%define _rpmfilename %%{ARCH}/%%{NAME}-v%%{VERSION}-%%{ARCH}.rpm
+
+%define name saf
+%define release 1
```

Comment:
I see that the built RPM has the right semver on it, but will hardcoding the release in the .spec file have consequences?

Reply:
No.  We do not push out releases without updating the version number.  Even trying to do so is fundamentally incompatible with npmjs (ex. the errors we see in the workflow to push the libs and applications to npmjs when a version change has not occurred in inspecjs usually but sometimes the other ones too) so I figured I'd make life simpler here as well.